Challenge1-Vite/src/components/Card.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+interface CardProps {
+  product: {
+    id: number;
+    thumbnail: string;
+    title: string;
+    description: string;
+    price: number;
+  };
+  favorites: Set<number>;
+  toggleFavorite: (id: number) => void;
+}
+
+const Card: React.FC<CardProps> = ({ product, favorites, toggleFavorite }) => {
```

Comment:
Maybe we can rename this to ProductCard which better reflects what the component is